src/styles/starred.scss

```diff
@@ -0,0 +1,28 @@
+.starred {
```

Comment:
Consider moving component styled to the corresponding component's directory for a better project structure, making it easier to find at the same time.